srcpkgs/audit/files/auditctl/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+test ! -r /etc/default/auditctl || . /etc/default/auditctl
+
+case "$USE_AUGENRULES" in
```

Comment:
i suppose it would be nice if it could source 'conf' like it is done on wpa_supplicant to get this value